functional/debug_console/run.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+SCRIPT_PATH=$(dirname "$0")
+
+source "${SCRIPT_PATH}/../../lib/common.bash"
+
+extract_kata_env
+if [ -z "${INITRD_PATH}" ]; then
+	info "Skip: initrd path is empty"
+	exit 0
+fi
+
+
+info "Enable debug console"
+sudo cp "${RUNTIME_CONFIG_PATH}" "${RUNTIME_CONFIG_PATH}.old"
+
+info "Disable debug"
+sudo sed -i \
+	 -e 's/^#*\(kernel_params\).*=.*$/\1 = \"agent.debug_console\"/g' "${RUNTIME_CONFIG_PATH}" \
+	 -e 's/^\(enable_debug.*\)/#\1/g' \
+	 "${RUNTIME_CONFIG_PATH}"
+
+info "Running container"
+cont_name=debugconsole
+docker run --net=none --name=${cont_name} --runtime=${RUNTIME} -dti busybox sh
+
+console_path="$(ps aux | grep "${HYPERVISOR_PATH}" | grep -E -o "/run/vc/vm/[a-z0-9]+/console.sock")"
```

Comment:
OOI, is there any more formal way to find such info, maybe from a `kata-runtime list` or similar? Just curious - although in theory this would hopefully work (given there are no other stuck VMs hanging around), it feels a bit fragile.